### DISCUSSION TITLE
Sun-2000 standby mode - not charging battery

### DISCUSSION THREAD

**Discussion by zapsterdk** [wlcrs/huawei_solar#490] [Setting up the integration]
Hi all,

Am I missing something here?

At night I want to charge my battery. But it seems like the inverter have changed to "stand-by no sunlight" mode. And it looks like I can't charge the battery due to this.

Are there any way to wake the inverter to Active mode so I can charge the battery?

Best Regards
Raymond

**Comment by Roving-Ronin** [comment]
@zapsterdk 
What mode of operation do you have the LUNA ESS configured to operate with ?
See this link for examples, in particular the ToU mode : https://photomate.zendesk.com/hc/en-gb/articles/4905182990109-Application-Scenarios-for-ESS-LUNA2000

Also what version of firmware are you running on inverters, luna ess and the MBUS (seperate update for modbus on each of these devices)? 
I forget the specific version number, but there was an update maybe 4 months ago, that added a lot of features/options in regards operating modes for the LUNA ESS (including the ToU and ability to enter up to 14 time periods for it), that is worth updating for. 

Update via FusionSolar application, but if you want to access the release notes this site is handy for accessing them in one place: 
https://photomate.zendesk.com/hc/en-gb/categories/4406398674589
